wagtail/contrib/settings/tests/generic/test_admin.py

```diff
@@ -88,6 +88,9 @@ def test_get_edit(self):
         response = self.get()
         self.assertEqual(response.status_code, 200)
 
+        with self.assertNumQueries(12):
```

Comment:
suggestion: add note as to what those are. I feel that we have a few "magic numbers" like that around.